docs/developers/package-dev/package-tutorial-from-scratch.md

```diff
@@ -0,0 +1,53 @@
+# Creating and Publishing a DAppNode Package with DAppNodeSDK
+
+DAppNodeSDK is a tool that simplifies the process of creating and publishing a DAppNode package (DNP). It generates necessary files for the package, validates its contents, and facilitates uploading and publishing it to [IPFS](https://ipfs.tech/). In this guide, we will show you how to use DAppNodeSDK to develop a DNP.
+
+Before proceeding, it is assumed that you have a basic understanding of DNP and the DAppNode ecosystem. If you're new to DAppNode, we recommend starting with the official [User Guide](https://docs.dappnode.io/).
+
+If you are someone who prefers following videos, scroll down for a 1h workshop on how to build packages.
+
+## Prerequisites
+1. [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+## Installing DAppNodeSDK
+To install DAppNodeSDK, open a terminal window and run the following command:
+
+```console
+$ npm install -g @dappnode/dappnodesdk
+```
+This will install DAppNodeSDK globally on your machine.
```

Comment:
You could add that running `npx @dappnode/dappnodesdk <command>` will ensure that you are using the latest SDK version and it can avoid the need of installing it globally.

Reply:
added, thanks!